### PR TITLE
LegacyStringToItemParser ➜ StringToItemParser

### LINKED ISSUE #15: Use item ``string name`` instead ``ID``
For the next PMMP updates (``LegacyStringToItemParser`` will be deprecated).

### PR DESCRIPTION
**Pull request by Steellgold**
Close #15

### CODE REVIEW

File: plugin.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: ChestFinder
 main: bluzzi\chestfinder\Main
 author: Bluzzi
-version: 2.0.0
+version: 2.0.1
```
Comment (Bluzzi) [inline plugin.yml]: 2.1.0 for a change that breaks compatibility

File: resources/config.yml [hunk 2]
```diff
@@ -24,4 +24,4 @@ detection:
 repeat: 1
 
 # Plugin's version, please don't touch it !
-version: "2.0.0"
\ No newline at end of file
+version: "2.0.1"
```
Comment (Bluzzi) [inline resources/config.yml]: 2.1.0 for a change that breaks compatibility

**Comment by Bluzzi** [comment]
- [x] It is also necessary to make the change in the task

**Comment by Steellgold** [comment]
> * [x]  It is also necessary to make the change in the task

Fixed: https://github.com/Symphonia-Team/ChestFinder/pull/16/commits/00417fcb17d1b023490c71bf6015e7f1766bedc2

File: resources/config.yml (hunk 2 above)
Comment (Steellgold) [reply resources/config.yml]: Fixed: https://github.com/Symphonia-Team/ChestFinder/pull/16/commits/53938e81886affe447be2a5340182f975be9f44f

File: plugin.yml (hunk 1 above)
Comment (Steellgold) [reply plugin.yml]: Fixed: https://github.com/Symphonia-Team/ChestFinder/pull/16/commits/df2d7ea74007c207d1d488292f5ec20dce9d1ce2

**Pull request merged**